webpack.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   output: {
     path: path.resolve(__dirname, "dist"),
     filename: "[name].bundle.js",
+    publicPath: "/",
```

Comment:
Becareful - this is limited to Unix-like systems I believe? This is why I have all the `path.resolve(__dirname, ...)` laying round.

Reply:
I'll try to change it. However, will `path.resolve(__dirname)` achieve the same effect as `"/"`? I'm not sure that if the server is unix-like, or even if the server is windows, `"/"` might still work? Because I guess URL standard is using `"/"`.

Reply:
Is `publicPath` supposed to be a path *relative* or *absolute* to the project's working directory? Because `__dirname` gives you the platform-dependent/platform-normalized *absolute* path to the project's **working directory**, and so [`Path.resolve("/")`](https://nodejs.org/api/path.html#path_path_resolve_paths) *should* be equivalent to `__dirname`.

But I think the this `"/"` almost certainly is only valid on Unix-like systems, because under Windows it will need to be `"\"`. If `publicPath` can accept an *absolute* path, then just assigning `__dirname` to it should be fine I think?

Reply:
It seems `path.resolve(__dirname, "/")` works, but not `path.resolve(__dirname)`. Will this still be a problem on non-unix systems though?

Reply:
If you look at the docs for `Path.resolve`, the `Path` API is designed exactly to handle platform-independent paths  In fact, I believe `resolve` will construct a platform-normalized path to the project's WD.

Reply:
Ohhhh I missed your reply (why isn't github doing its hot reload correctly lol?), yes `Path.resolve("/")` works and is the intended way (absolute path). So now it should work.

Reply:
> It seems `path.resolve(__dirname, "/")` works, but not ...

This also makes sense now as it is eventually resolving the path to "/" which is equivalent to `Path.resolve("/")`.